### PR TITLE
add is_sass_environment var

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

File: corehq/util/context_processors.py [hunk 1]
```diff
@@ -91,7 +91,8 @@ def websockets_override(request):
 
 def enterprise_mode(request):
     return {
-        'enterprise_mode': settings.ENTERPRISE_MODE
+        'enterprise_mode': settings.ENTERPRISE_MODE,
+        'is_sass_environment':  settings.IS_SASS_ENVIRONMENT
```
Comment (dimagimon) [inline corehq/util/context_processors.py]: E241 multiple spaces after ':'

**Comment by snopoke** [comment]
bump

**Review by sravfeyn — APPROVED** [review]
Looks good, was waiting for product confirmation on the email thread. Feel free to self-merge otherwise

**Review by mkangia — APPROVED** [review]

File: settings.py [hunk 2]
```diff
@@ -923,6 +923,8 @@
 
 _location = lambda x: os.path.join(FILEPATH, x)
 
+IS_SASS_ENVIRONMENT = SERVER_ENVIRONMENT == 'production'
```
Comment (benrudolph) [inline settings.py]: what is this for?

Comment (snopoke) [inline settings.py]: just 'is this production' i.e. do we want features like 'live chat'

Comment (benrudolph) [inline settings.py]: ohh i thought this was in reference to SASS: http://sass-lang.com/guide

Comment (orangejenny) [inline settings.py]: ha, I thought this too...maybe a different name would be good. what does "sass" mean in this context?

**Comment by snopoke** [comment]
Sorry, wasn't paying much attention to this - it was meant to be `saas`

**Pull request merged**